jjb/clm/templates/clm-rust-templates.yaml

```diff
@@ -0,0 +1,189 @@
+####################
+# COMMON FUNCTIONS #
+####################
+
+- lf_rust_common: &lf_rust_common
```

Comment:
We can actually remove this lf_rust_common block and just re-use "lf_maven_common" . I noticed they are same.